### DISCUSSION TITLE
How far are we from production?

### DISCUSSION THREAD

**Discussion by francisluz** [input-output-hk/cardano-js-sdk#111] [General]
Hi Team,

I'm from @B58-Finance wallet, and we would like to use some of the components in production. the packages that have been already released are stable enough?

Awesome work you guys have done so, our internal lib looks pretty similar in terms of concept.

**Comment by rhyslbw** [comment]
Hi @francisluz, thanks for reaching out.

Without committing to a firm release date, we are expecting to be making a 1.0 release before EOY. There will however be beta releases that will have production-grade QA, but there may be some breaking changes based on feedback from real-world use.

> the packages that have been already released are stable enough?

It's not recommended at this point as there are some important features, such as https://github.com/input-output-hk/cardano-js-sdk/pull/112,  still in development. The current development velocity should give you a target within a couple of weeks to start considering integration in a non-production environment.

> Awesome work you guys have done so, our internal lib looks pretty similar in terms of concept.

cc @mkazlauskas. Thank you! We're open to feedback and suggestions on API design, features, or other gaps you may see.

**Reply by francisluz** [reply]
Got it @rhyslbw, awesome to know, we will release our wallet before that.

One specific feature that we were looking at and seems that you guys have already done is the Coin Selection Algo. and the dApp interface, which is the same that we're going to use.

As a beta we're ok with breaking changes. So, we won't rely on the SDK for now, but there's some interesting work done from you guys so far. We'll open source our code as soon as we release the Beta.

Thanks
